notebooks/Generate_Tables.ipynb

```diff
@@ -0,0 +1,71 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to Generate Source and Object Tables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lenspop\n",
+    "import om10\n",
+    "import desc.slrealizer\n",
+    "import format_sdss_to_om10\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# initialize om10\n",
+    "db = om10.DB(vb=False, catalog='../../../data/qso_mock.fits')\n",
+    "# we select LSST-like lenses by quering with these values\n",
+    "db.select_random(maglim=23.3,area=5000.0,IQ=0.75)\n",
+    "# calculate the synthetic magnitude for the OM10 catalog -- need this\n",
+    "db.paint(synthetic=True)\n",
+    "# calculate the sizes for the OM10 catalog -- if you want to assume the point-source-like-galaxies, get rid of this\n",
+    "#db.calculate_size()\n",
+    "# initialize realizer\n",
+    "realizer = desc.slrealizer.SLRealizer(catalog=db, observation='../../../data/twinkles_observation_history.csv')\n",
+    "t0 = time.time()\n",
+    "realizer.make_source_catalog(save_dir='../../../data/source_catalog_galsim_noise_perfect.csv')\n",
```

Comment:
Shouldn't the `save_dir` kwarg actually be called `save_file`?

Reply:
True! Fixed in the source code (deleting notebook).